test/mock/MockERC20Metadata.sol

```diff
@@ -0,0 +1,18 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity >=0.8.25 <0.9.0;
+
+// solhint-disable custom-errors,max-line-length
+import { ERC20 } from "@openzeppelin/contracts/token/ERC20/ERC20.sol";
+
+contract MockERC20Metadata is ERC20 {
```

Comment:
As above.

Reply:
Move this contract to `TestERC20.sol`. That file has multiple TestERC20 contracts